src/osx/cocoa/renderer.mm

```diff
@@ -162,13 +192,62 @@ wxRendererNative& wxRendererNative::GetDefault()
     return s_rendererMac;
 }
 
+wxRendererMac::wxRendererMac()
+{
+#if wxOSX_USE_NSCELL_RENDERER
+    m_nsPushButtonCell = [[NSButtonCell alloc] initTextCell:@""];
```

Comment:
Do you think it could be worth to allocate all these cells on demand only, i.e. only when a renderer method using it is called for the first time? Or are cells so cheap to create that it's not worth optimizing this?

I'm just a bit wary of adding more and more things unconditionally done on every application startup, each of them doesn't cost much but sooner or later they're going to add up to something noticeable.

Reply:
The CPU on my M1 is 6 msec for the allocating all cells, so at the moment I'd rather leave it like that, if that time is ok

Reply:
To be honest, 6ms doesn't seem negligible to me: I've just tested and with the debug build of wxGTK (I was too lazy to rebuild in release, but it should be smaller there) the time-to-first-paint is 60ms, i.e. it's 10% of the total startup time.

If you'd like, I can make the changes to create them on demand, but I'd just like to ask you to test them because I'm not sure how to do it.

Reply:
> If you'd like, I can make the changes to create them on demand, but I'd just like to ask you to test them because I'm not sure how to do it.

@vadz I'd very much appreciate that, I'll test it, thanks :-)

Reply:
I've pushed my changes now (includes a warning fix I was getting here), please let me know if I did anything stupid. TIA!